packaging/opensuse/snapd.spec

```diff
@@ -118,12 +142,15 @@ CXXFLAGS="$RPM_OPT_FLAGS -fPIC -Wl,-z,relro -Wl,-z,now"
 export CFLAGS
 export CXXFLAGS
 
-# NOTE: until snapd and snap-confine have the improved communication mechanism
-# we need to disable apparmor as snapd doesn't yet support the version of
-# apparmor kernel available in SUSE and Debian. The generated apparmor profiles
-# cannot be loaded into a vanilla kernel. As a temporary measure we just switch
-# it all off.
-%configure --disable-apparmor --libexecdir=%{_libexecdir}/snapd
+# N.B.: Prior to openSUSE Tumbleweed in May 2018, the AppArmor userspace in SUSE
+# did not support what we needed to be able to turn on basic integration.
+%configure \
+    %{!?with_apparmor:--disable-apparmor} \
```

Comment:
Makes me wonder if we should have an explicit --enable-apparmor switch rather than rely on autodetection for the case when we want it supported?

Reply:
@bboozzoo, It'd be nicer if it did, but since snapd assumes Ubuntu AppArmor anyway, it's probably not going to happen.

Maybe it would if snapd gained some SELinux integration leveraging `libselinux`, it might happen, because then you'd want to specifically enable what MACs your distro ships with support for.